plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -287,24 +287,15 @@
             modes="run,debug">
          <contextualLaunch>
             <enablement>
-              <or>
-                  <with variable="selection">
-                     <iterate>
-                        <adapt type="org.eclipse.core.resources.IProject">
-                           <test
-                                 property="org.eclipse.wst.common.project.facet.core.projectFacet"
-                                 value="com.google.cloud.tools.eclipse.appengine.facets.standard" />
-                           </adapt>
-                     </iterate>
-                  </with>
-                  <with variable="activeEditorInput">
-                     <adapt type="org.eclipse.core.resources.IFile">
+               <with variable="selection">
```

Comment:
We can drop this outer `<with variable="selection">` and just use the `<iterate>…</iterate>` on value that's passed in (that is likely — but not necessarily — the selection).